build/commands/lib/l10nUtil.js

```diff
@@ -210,43 +210,6 @@ const l10nUtil = {
     return l10nUtil.getAllBravePaths().filter((x) => ['grd', 'json'].includes(x.split('.').pop()))
   },
 
-// Helper function to retrieve Greaselion script paths relative to the
-// Brave paths.
-//
-// Greaselion.json consists of an array of Greaselion rules,
-// specifying scripts to inject into given sites based on certain
-// preconditions. If the rule contains a "messages" key, then the
-// script contains user-visible strings that require translation. This
-// helper function gathers those messages.json files for transmission
-// to Transifex.
-  getGreaselionScriptPaths: (extensionPath) => {
-    let basePath = extensionPath
-    if (!basePath) {
-      basePath = '../../../brave-site-specific-scripts'
-    }
-
-    const jsonContent = fs.readFileSync(`${basePath}/Greaselion.json`, 'utf8')
-    if (!jsonContent) {
-      console.error('Missing Greaselion.json')
-      return []
-    }
-
-    const greaselionRules = JSON.parse(jsonContent)
-    if (!greaselionRules) {
-      console.error('Malformed Greaselion.json')
-      return []
-    }
-
-    let paths = []
-    greaselionRules.forEach((rule) => {
-      if (rule.messages) {
```

Comment:
@mkarolin @zenparsing FIY
I can't see any `messages` here: https://github.com/brave/brave-site-specific-scripts/blob/master/Greaselion.json

Reply:
We removed the scripts that used that feature, from what I recall. We can probably remove the Greaselion section from Transifex if it's still there @mkarolin.